src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,52 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_LIMIT = 10;
+    private static final String OUT_OF_LIMIT_EXCEPTION = "The storage is full";
+    private final K[] keys;
+    private final V[] values;
+    private int currentStorageSize = 0;
```

Comment:
Better to name it size
```suggestion
    private int size = 0;
```